django/db/models/manager.py

```diff
@@ -23,15 +22,12 @@ def ensure_default_manager(cls):
         setattr(cls, 'objects', SwappedManagerDescriptor(cls))
         return
     if not getattr(cls, '_default_manager', None):
-        # Create the default manager, if needed.
-        try:
-            cls._meta.get_field('objects')
+        if any(f.name == 'objects' for f in cls._meta.fields):
```

Comment:
Is this change needed? I'm wondering why iterating over `fields` is preferred to `get_field()` as it seems like the new way would be slower and is a bit less explicit IMO.